cluster/gce/gci/configure-helper.sh

```diff
@@ -449,15 +449,19 @@ function ensure-local-ssds-ephemeral-storage() {
     return
   fi
 
-  # Create RAID array
-  local md_device=(/dev/md/*)
-  md_device=${md_device[0]}
-  if [ ! -e "$md_device" ]; then
-    md_device="/dev/md/0"
-    echo "y" | mdadm --create "${md_device}" --level=0 --raid-devices=${#devices[@]} ${devices[@]}
+  local device="${devices[0]}"
+  if [ "${#devices[@]}" -ne 1 ]; then
+    local device=(/dev/md/*)
```

Comment:
Is this going to set a new variable locally, and not modify the one at L452?

Reply:
Oddly enough, it worked in manual testing. Fixed.